packages/oui-pagination/src/index.spec.js

```diff
@@ -302,6 +305,9 @@ describe("ouiPagination", () => {
                     </oui-pagination>
                 `);
 
+                // Page sizes list should not list page size above pageSizeMax (50).
```

Comment:
I understand better with that comments thanks!
IMHO I think if you rename your tests, you will not need big comments

maxPageSize = 50
- Should display sizes lower or equal to max page size in page size list
//Page size list = [25, 50, 100, 200]
expect(pageSizesButtons).toEqual([25, maxPageSize]); (it's more meaningfull than length 2)

- Should add max page size value to the page size list if it doesn't exist
//Page size list = [25, 50]
expect(pageSizesButtons).toEqual([25, 50, maxPageSize]);